dev/Scroller/Scroller.h

```diff
@@ -775,18 +807,18 @@ class Scroller :
     winrt::ICoreWindow::KeyUp_revoker m_coreWindowKeyUpRevoker{};
 
     //Bug 16792535: VectorChanged_revoker does not currently work.
```

Comment:
>//Bug 16792535: VectorChanged_revoker does not currently work. [](start = 4, length = 62)

I Believe this has been fixed, might be worth changing to use the auto revoker

Reply:
Done.